outdated/__init__.py

```diff
@@ -36,7 +36,7 @@ def check_outdated(package, version):
                 latest = None
 
     def get_latest():
-        url = 'https://pypi.python.org/pypi/%s/json' % package
+        url = (repository_url + "%s/json") % package
```

Comment:
Please:

1. Make it work whether or not `repository_url` ends with a slash.
2. Don't use `+` since there's already string interpolation.